src/popup/router/index.ts

```diff
@@ -69,7 +69,9 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
-  if (!store.getters['sdkPlugin/sdk'] && !RUNNING_IN_POPUP) initSdk();
+  const { isSdkReady } = useSdk13({ store });
+
+  if (!isSdkReady && !RUNNING_IN_POPUP) initSdk();
```

Comment:
I think this needs to be removed as the logic of the SDK initialization is totally different than when this lines was added to the codebase.